addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -154,6 +157,11 @@ FormRenderer.include({
             await this._mountChatterContainerComponent();
         }
     },
+    _reloadRecord() {
+        this.trigger_up('discard_changes', {
```

Comment:
Btw I don't think we can do it here, but in v13 even the new message itself is not posted if the user select to not refresh.

Reply:
It's feasible, but it's more hacky (we'd have to trigger the discard_changes from within the composer, which means now we need to know - in the composer - whether the form is going to need a refresh or not).  And questionable whether it'd be desirable.